src/config.template.yaml

```diff
@@ -132,8 +132,6 @@ platforms:
     min_seconds_to_record: 3
 
   dummy:
-    # whether to print messages when methods are called
-    verbose: false
```

Comment:
I know that `dummy` don't use config, but for consistency each platform must expect that it's config is a `dict`. If you remove "verbose", dummy will receive `str` instead of `dict`. Better to put here explicit empty `dict`:

```
dummy: {}
```